package/network/services/hostapd/files/hostapd.uc

```diff
@@ -280,10 +281,14 @@ function find_array_idx(arr, key, val)
 
 function bss_reload_psk(bss, config, old_config)
 {
-	if (is_equal(old_config.hash.wpa_psk_file, config.hash.wpa_psk_file))
```

Comment:
This shouldn't be here, it should instead regenerate the config if the sae_password_file is different. This won't do anything.

Reply:
I am really not sure about this one but simply put: if it doesn't do anything here, why would it do something in case of change to `wpa_psk_file` ?  Shouldn't the logic be the same for both `sae_password_file` and `wpa_psk_file` ?

Reply:
Because `RELOAD_WPA_PSK` only reloads the wpa_psk_file and it's functionality handled by hostapd itself, `sae_password_file` is not handled by hostapd (as we're generating the sae_password line on the fly) and so we will need to regenerate the config and reload if it changed.

Reply:
Commit https://github.com/openwrt/openwrt/pull/14448/commits/b78f011fc09da040a03f038ff4a0b1fb811a51fa is my best guess at solving this one. I never did touch UnrealScript before and so I cannot guarantee syntax is correct but the logic should work

Reply:
It's not UnrealScript, it's ucode: https://github.com/jow-/ucode

Reply:
> It's not UnrealScript, it's ucode: https://github.com/jow-/ucode

As you can imagine I did never used ucode either, thanks :)